### PR TITLE
Document 'Cmake.get_version()'

### PR DESCRIPTION
**Pull request by jgsogo**
Docs for https://github.com/conan-io/conan/pull/3586

### CODE REVIEW

File: reference/build_helpers/cmake.rst [hunk 1]
```diff
@@ -305,6 +305,20 @@ example:
...
+.. code-block:: python
+
+    @staticmethod
+    def get_version()
+
+
+Returns the CMake version in a ``conans.model.Version`` object as it is evaluated by the
+command line. Will raise if cannot resolve it to valid version.
+
+
```
[the remark this answers is not in the document]
Comment (lasote) [reply reference/build_helpers/cmake.rst]: Write an example of comparison, you can compare the return with a string directly, not needed to instance a Version.

[the remark this answers is not in the document]
Comment (jgsogo) [reply reference/build_helpers/cmake.rst]: @danimtb I would keep this blank line, the next section is a higher order one and I think that this extra blank line helps.

@lasote This is not the place to show how to work with a `Version` object, maybe we should document the class itself and show examples somewhere else (and link from here).

[the remark this answers is not in the document]
Comment (lasote) [reply reference/build_helpers/cmake.rst]: Maybe a "misc" in the reference? where we could put ConanFileReference, Version, and others.

[the remark this answers is not in the document]
Comment (danimtb) [reply reference/build_helpers/cmake.rst]: @jgsogo ok
@lasote I like the idea but we could improve the naming

**Pull request merged**